src/style.css

```
@@ -1 +1,9 @@
-/* styles go here */
+.container {
+  margin: 0 2px 20px;
+  padding: 1px 12px 5px;
+}
+
+.block {
+  display: block;
+  padding: 5px 0;
+}
```

some tests failed. 
suggest some refactor in styles, it can help to fix tests
```suggestion
.container {
  margin-bottom: 20px;
  padding-bottom: 0;
}

.block {
  display: block;
  padding-bottom: 10px;
}
```

---

Thanks. Done